tools/bundle-size-tools/src/compareBundles.ts

```diff
@@ -37,3 +37,22 @@ export function compareBundles(baseline: BundleSummaries, compare: BundleSummari
 
   return results;
 }
+
+/**
+ * Checks if a bundle comparison contains no size changes
+ * @param comparisons
+ */
+export function bundlesContainNoChanges(comparisons: BundleComparison[]): boolean {
+  for (let i = 0; i < comparisons.length; i++) {
```

Comment:
_non-functional_

Thoughts?
`for (const { commonBundleMetrics } of comparisons) {`

And below:
```
const metrics = Object.values(commonBundleMetrics); // values ignores keys
for (const { baseline, compare } of metrics) {
```

Or an unreadable(?) one-liner:
```typescript
comparisons.every(({ commonBundleMetrics }) => Object.values(commonBundleMetrics)
    .every(({ baseline, compare }) => baseline.parsedSize === compare.parsedSize));
```